.gitignore

```diff
@@ -12,6 +12,7 @@
 
 # misc
 .idea
+.tern-port
```

Comment:
I generally prefer to keep ide/editor/os specific ignores out of project .gitignore, as it can proliferate. You can maintain your own global ignore file: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Ah, visste ikke om dette! Fjerner endringen.